x-pack/plugins/security_solution/public/resolver/models/process_event.ts

```diff
@@ -144,7 +144,8 @@ export function processPath(passedEvent: ResolverEvent): string | undefined {
  */
 export function userInfoForProcess(
   passedEvent: ResolverEvent
-): { user?: string; domain?: string } | undefined {
+  // TODO, fix in 7.9
+): { name?: string; domain?: string } | undefined {
```

Comment:
@james-elastic seems like these values are wrong (and wont work in the UI) in 7.9. not a 100% but we should look into it

Reply:
TODO remove `TODO`

Reply:
The change you're proposing is correct. The data that is sent by the endpoint looks like this:

</details><summary>Endpoint process event</summary>

```
{
  ...
"event": {
    "sequence": 81869,
    "ingested": "2020-07-28T18:10:08.032404Z",
    "created": "2020-07-28T18:07:14.30791200Z",
    "kind": "event",
    "module": "endpoint",
    "action": "end",
    "id": "LlVyJJ01mAI5jUhA+++/7VTY",
    "category": [
      "process"
    ],
    "type": [
      "end"
    ],
    "dataset": "endpoint.events.process"
  },
  "dataset": {
    "name": "endpoint.events.process",
    "namespace": "default",
    "type": "logs"
  },
  "user": { <--------------------------------
    "domain": "NT AUTHORITY",
    "name": "LOCAL SERVICE"
  },
  "_index": ".ds-logs-endpoint.events.process-default-000001",
  "_type": "_doc",
  "_id": "ZpqelnMBlglBfzvyECQh",
  "_score": 1
}
```

</details>

The fields that are defined in the mapping are `user.id`, `user.name`, and `user.domain`: https://github.com/elastic/endpoint-package/blob/master/schemas/v1/process/process.yaml#L1260

Reply:
@oatkiller @jonathan-buttner  Should pull this (and the other user fix) out into its own PR to get it in the BC today?